src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+logging.level.org.hibernate=ERROR
+logging.path=logs
```

Comment:
Where does the default log to? Sysout? Why nto use that as the default?

Reply:
Adding logging.path doesn't affect default logging. It'll be available anyway, but additionally, there will be a file, to see the logs later.